terraform/infrastructure/samples/aws/data.tf

```diff
@@ -0,0 +1,44 @@
+data "aws_vpc" "current_vpc" {
+  filter {
+    name   = "tag:Name"
+    values = ["<CLUSTER_NAME>-vpc"]
+  }
+}
+
+# data "aws_subnets" "intra_subnets" {
+#   filter {
+#     name   = "vpc-id"
+#     values = [data.aws_vpc.vpc_id]
+#   }
+#   tags = {
+#     Tier = "Intra"
+#   }
+# }
+
+data "aws_subnet" "intra_subnet_1a" {
+  filter {
+    name   = "tag:Name"
+    values = ["<CLUSTER_NAME>-vpc-intra-eu-west-1a"]
```

Comment:
please change eu-west-1 to the parameter <CLOUD_REGION> here and in two other subnets bellow

Reply:
done